bouncer/blacklist/views.py

```diff
@@ -9,8 +10,34 @@
 )
 
 
-class IPRequestView(View):
+class RequestView(View):
+    template = loader.get_template("blacklist/request.html")
+
     def get(self, request, *args, **kwargs):
-        if is_ip_blacklisted(request.GET["ip"]):
-            return HttpResponse("YES")
-        return HttpResponse("NO")
+        query = dict(self.request.GET.lists())
+        context = {"results": []}
+        if "email" in self.request.GET:
+            while query["email"] != []:
```

Comment:
There's a better way to loop through an array in Python 😇

```python
for item in array:
  # do stuff
```

Please change the loop to make it more readable.